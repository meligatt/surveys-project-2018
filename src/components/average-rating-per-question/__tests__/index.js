import React from 'react';
import renderer from 'react-test-renderer';
import AverageRatingPerQuestion from '../index';

describe('AverageRatingPerQuestion', () => {
  it('renders correctly', () => {
    const themesMock = [{
        name : "theme 1",
        questions: [],
      },
      {
        name : "theme 2",
        questions: [],
      }];

    const tree = renderer.create(
      <AverageRatingPerQuestion themes = {themesMock} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('renders null when empty themes is passed as prop', () => {
    const themesMock = [];

    const tree = renderer.create(
      <AverageRatingPerQuestion themes = {themesMock} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
