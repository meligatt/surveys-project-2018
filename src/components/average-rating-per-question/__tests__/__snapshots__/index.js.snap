// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AverageRatingPerQuestion renders correctly 1`] = `
<div>
  <div>
    <p>
      the average rating (from 1 to 5) for each question
    </p>
  </div>
</div>
`;

exports[`AverageRatingPerQuestion renders null when empty themes is passed as prop 1`] = `null`;
